pkg/app/piped/livestatestore/kubernetes/store.go

```diff
@@ -69,6 +69,11 @@ func (s *store) initialize() {
 			continue
 		}
 
+		// Ignore in case appNodes with appID not existed in store.
+		if s.apps[appID] == nil {
```

Comment:
Because this is an unexpected situation so can you add a log here to help us figure out what resource is causing this problem?